dozer-integrations/dozer-proto3/pom.xml

```diff
@@ -22,15 +22,14 @@
 
     <modelVersion>4.0.0</modelVersion>
     <parent>
+        <artifactId>dozer-integrations</artifactId>
         <groupId>com.github.dozermapper</groupId>
-        <artifactId>dozer-plugins-parent</artifactId>
         <version>6.2.0-SNAPSHOT</version>
-        <relativePath>../plugins-parent</relativePath>
```

Comment:
Can you add relativePath for dozer-integrations

Reply:
What for? `dozer-integrations` is the direct parent of current project.